.github/workflows/release.yml

```diff
@@ -0,0 +1,198 @@
+# **what?**
+# Take the given commit, run unit tests specifically on that sha, build and 
+# package it, and then release to GitHub and PyPi with that specific build
+
+# **why?**
+# Ensure an automated and tested release process
+
+# **when?**
+# This will only run manually with a given sha and version
+
+name: Release to GitHub and PyPi
+
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+       description: 'The last commit sha in the release'
+       required: true
+      version_number:
+       description: 'The release version number (i.e. 1.0.0b1)'
+       required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  unit:
+    name: Unit test
+
+    runs-on: ubuntu-latest
+
+    env:
+      TOXENV: "unit"
```

Comment:
Is unit testing enough verification for moving forward with the release or should we run some integration test? We should only be releasing when the last commit is all 🟢 in Actions but that relies on us doing our due diligence and checking

Reply:
Unit tests should catch the very obvious issues. Of course, if there are failing integration tests on the `main`/release branch, we should know about them